File: src/main/java/xin/vanilla/entity/instruction/KanriInstructions.java
package xin.vanilla.entity.instruction;

import java.util.HashSet;
import java.util.Set;

/**
 * 群管操作指令
 */
public class KanriInstructions {
    // 群管指令前缀-
    public String prefix = "";

    // 群管理+
    public Set<String> admin = new HashSet<String>() {{
        add("ad");
    }};

    // 头衔+
    public Set<String> tag = new HashSet<String>() {{
        add("tag");
    }};

    // 群名片+
    public Set<String> card = new HashSet<String>() {{
        add("card");
    }};

    // 戳一戳+
    public Set<String> tap = new HashSet<String>() {{
        add("tap");
        add("slap");
    }};

    // 禁言+
    public Set<String> mute = new HashSet<String>() {{
        add("mute");
        add("ban");
    }};

    // 解除禁言+
    public Set<String> loud = new HashSet<String>() {{
        add("loud");
    }};

    // 撤回+
    public Set<String> withdraw = new HashSet<String>() {{
        add("recall");
        add("withdraw");
        add("rec");
    }};

    // 踢出(危险操作, 使用特殊语法)*
    public String kick = "kick [VA_CODE.QQS] out";

    // 精华消息+
    public Set<String> essence = new HashSet<String>() {{
        add("essence");
        add("fine");
    }};
}

File: src/main/java/xin/vanilla/entity/config/instruction/KanriInstructions.kt
package xin.vanilla.entity.config.instruction

import kotlinx.serialization.Serializable
import lombok.Data
import lombok.experimental.Accessors

/**
 * 群管操作指令
 */
@Data
@Accessors(chain = true)
@Serializable
class KanriInstructions {
    /**
     * 群管指令前缀-
     */
    var prefix: String = ""

    /**
     * 头衔+
     */
    var tag: Set<String> = mutableSetOf("tag")

    /**
     * 群名片+
     */
    var card: Set<String> = mutableSetOf("card")

    /**
     * 戳一戳+
     */
    var tap: Set<String> = mutableSetOf("tap", "slap")

    /**
     * 禁言+
     */
    var mute: Set<String> = mutableSetOf("mute", "ban")

    /**
     * 解除禁言+
     */
    var loud: Set<String> = mutableSetOf("loud")

    /**
     * 撤回+
     */
    var withdraw: Set<String> = mutableSetOf("recall", "withdraw", "rec")

    /**
     * 踢出(危险操作, 使用特殊语法)*
     */
    var kick: String = "kick [VA_CODE.QQS] out"

    /**
     * 精华消息+
     */
    var essence: Set<String> = mutableSetOf("essence", "fine")

    /**
     * 群管理+
     */
    var admin: Set<String> = mutableSetOf("groupAdmin", "gad", "admin", "ad")

    /**
     * 副管+
     */
    var deputyAdmin: Set<String> = mutableSetOf("deputyAdmin", "deputyAd", "dad")

    /**
     * 主管+
     */
    var botAdmin: Set<String> = mutableSetOf("botAdmin", "botAd", "bad")

    /**
     * 超管+
     */
    var superAdmin: Set<String> = mutableSetOf("superAdmin", "superAd", "sad")

    /**
     * 主人+
     */
    var botOwner: Set<String> = mutableSetOf("botOwner", "owner")
}
